model/user.py

```diff
@@ -17,6 +17,26 @@ def __init__(self, slack_id):
         self.__image_url = ""
         self.__permissions_level = Permissions.member
 
+    def get_attachment(self):
+        """Return slack-formatted attachment (dictionary) for user."""
+        text_pairs = [
+            ('Slack ID', self.__slack_id),
+            ('Name', self.__name),
```

Comment:
Maybe we should have a separate this into its own class, perhaps call it attachment generator class or something

Reply:
comment added